technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,72 @@
+# Continuous Profiling in TEAMMATES
+
+Authors: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Amrut Prabhu](https://github.com/amrut-prabhu) and [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [Overview of the Proposed Solution](#Overview-of-Solution)
+* [Tools considered for Performance Testing](#Tools-considered-for-Performance-Testing)
+* [Reasons for using JMeter](#Reasons-for-using-JMeter)
+* [Current implementation of the solution](#current-implementation-of-the-solution)
+* [Findings and Recommendations](#findings-and-recommendations)
+* [Future Work](#Future-work)
+
+## Introduction
+
+This report gives a brief overview of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes the reasons behind our proposed solution.
+
+## Problem
+
+TEAMMATES is one of the biggest student projects in the open source community. Currently, TEAMMATES boasts of a community comprising over 450 developers and a codebase 
+of nearly 130LoC. Maintaining such a project demands high quality standards to ensure long term survival. This means, 
```

Comment:
- 130 **K**LoC...
- Those numbers are only accurate at the time of writing. Mention it.
- Break at the sentence instead of abruptly at `of`, i.e.
  ```
  ... a codebase of nearly 130 KLoC.
  Maintaining such a ...
  ```